plugins/nginx-vhosts/commands

```diff
@@ -161,12 +161,24 @@ EOF
     TEMP_DIR=$(mktemp -d)
     cd $TEMP_DIR
     tar xvf - <&0
-    [[ ! -f "$TEMP_DIR/server.crt" ]] && echo "Tar archive missing server.crt" && exit 1
-    [[ ! -f "$TEMP_DIR/server.key" ]] && echo "Tar archive missing server.key" && exit 1
+
+    CRT_FILE_COUNT=$(ls -1 ./*.crt | wc -l)
+    if [[ $CRT_FILE_COUNT -lt 1 ]]; then
+      echo "Tar archive is missing .crt file" && exit 1
+    elif [[ $CRT_FILE_COUNT -gt 1 ]]; then
+      echo "Tar archive contains more than one .crt file" && exit 1
+    fi
+
+    KEY_FILE_COUNT=$(ls -1 ./*.key | wc -l)
+    if [[ $KEY_FILE_COUNT -lt 1 ]]; then
+      echo "Tar archive is missing .key file" && exit 1
+    elif [[ $KEY_FILE_COUNT -gt 1 ]]; then
+      echo "Tar archive contains more than one .key file" && exit 1
+    fi
 
     mkdir -p "$DOKKU_ROOT/$APP/tls"
-    mv "$TEMP_DIR/server.crt" "$DOKKU_ROOT/$APP/tls/server.crt"
-    mv "$TEMP_DIR/server.key" "$DOKKU_ROOT/$APP/tls/server.key"
+    mv "$TEMP_DIR/*.crt" "$DOKKU_ROOT/$APP/tls/server.crt"
+    mv "$TEMP_DIR/*.key" "$DOKKU_ROOT/$APP/tls/server.key"
```

Comment:
That's a great idea! `shellcheck` is telling me to use `find` in place of `ls` as well. I will implement both changes tomorrow.